lib/Runtime/Library/ConcatString.cpp

```diff
@@ -143,7 +143,9 @@ namespace Js
         this->propertyString = propStr;
         if (propStr != nullptr)
         {
-            this->propertyRecord = propStr->GetPropertyRecord();
+            Js::PropertyRecord const * localPropertyRecord;
+            propStr->GetPropertyRecord(&localPropertyRecord);
```

Comment:
Why the local variable here, rather than passing in `&(this->propertyRecord)`?

Reply:
That wouldn't trigger SWB operator

Reply:
Ah I see, makes sense. Could you please add a comment there so it doesn't get "cleaned up" later?